run_cdc_dev.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+# Runs an approximate custom dc setup for development.
+# Note that this is not the exact custom dc setup but one that can be easier to develop against vs doing it in a docker container.
+
+# Make sure the following are taken care of before running this script:
+# - run ./run_npm.sh
+# - run ./scripts/update_git_submodules.sh
+# - run ./run_test.sh --setup_python
+
+# Kill forked processes on exit.
+trap "trap - SIGTERM && kill -- -$$" SIGINT SIGTERM EXIT
+
+source .run_cdc_dev.env && export $(sed '/^#/d' .run_cdc_dev.env | cut -d= -f1)
```

Comment:
suggest to print out current website, mixer and import git hash with bright color